coreDriverInterfaceSupported.json

```diff
@@ -4,6 +4,7 @@
         "2.9",
         "2.10",
         "2.11",
-        "2.12"
+        "2.12",
+        "2.13"
```

Comment:
There is an associated array in the constants.py file as well. Please update that too (See the PR checklist, it has an item for this)

Reply:
Done. Thanks!